src/openforms/js/components/admin/form_design/registrations/camunda.js

```diff
@@ -0,0 +1,186 @@
+    let processDefinitions;
+    const { loading, value, error } = useAsync(
+        async () => {
+            const response = await get(PROCESS_DEFINITIONS_ENDPOINT);
+            if (!response.ok) throw new Error(`Response status: ${response.status}`);
+            return response.data;
+        }, [PROCESS_DEFINITIONS_ENDPOINT]
+    );
+
+    if (!loading && !error) {
+        // transform the process definitions in a grouped structure
+        processDefinitions = groupBy(value, 'key');
+    }
+
+    return {loading, processDefinitions, error};
+};
+
+
+// use rjsf wrapper to keep consistent markup/styling
+const Wrapper = ({children}) => (
+    <form className="rjsf" name="form.registrationBackendOptions">
+        <CustomFieldTemplate displayLabel={false} rawErrors={[]}>
+            <fieldset id="root">
+                {children}
+            </fieldset>
+        </CustomFieldTemplate>
+    </form>
+);
+
+
+const FormFields = ({processDefinitions, formData, onChange}) => {
+    const intl = useIntl();
+    const { processDefinition='', processDefinitionVersion=null } = formData;
+
+    const processDefinitionChoices = Object.entries(processDefinitions).map(([processKey, versions]) => {
+        // grab the first version name - it is theoretically possible the process name has changed in
+        // another version, but not much we can do about that
+        const name = versions[0].name;
+        return [processKey, `${name} (${processKey})`];
+    });
+
+    const versionChoices = !processDefinition
+        ? []
+        : processDefinitions[processDefinition].map(version => {
+            return [`${version.version}`, `v${version.version}`];
+        });
+
+
+    const onFieldChange = (event) => {
+        const {name, value} = event.target;
+        const updatedFormData = {...formData, [name]: value};
+
+        switch (name) {
+            // if the definition changes, reset the version
+            case 'processDefinition': {
+                updatedFormData.processDefinitionVersion = null;
+                break;
+            }
+            // normalize blank option to null
+            case 'processDefinitionVersion': {
+                if (value === '') {
+                    updatedFormData.processDefinitionVersion = null;
+                } else {
+                    updatedFormData.processDefinitionVersion = parseInt(value, 10);
+                }
+                break;
+            }
+        }
+
+        // call parent event-handler with fully updated form data object
+        onChange(updatedFormData);
+    };
+
+    return (
+        <Wrapper>
+            <CustomFieldTemplate
+                id="camundaOptions.processDefinition"
+                label={intl.formatMessage({
+                    defaultMessage: 'Process',
+                    description: 'Camunda \'process definition\' label'
+                })}
+                rawErrors={null} errors={null} // TODO
+                required
```

Comment:
The `processDefinition` is missing the help text (`rawDescription` field in the `processDefinitionVersion`)